sbpy/activity/gas/core.py

```diff
@@ -29,6 +29,7 @@
 
     'Haser',
     'Vectorial',
+    'GasComa'
```

Comment:
I left this out of __all__ because it is an abstract class that a user probably doesn't need.  What do you think?